### PR TITLE
06085 5367 cleanup genesis state2

### LINKED ISSUE #6085: Move genesis state construction out of the platform
simplifies workflow, we don't need to support a special genesis workflow

### PR DESCRIPTION
**Pull request by cody-littley**
Closes #6085

### CODE REVIEW

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/SwirldStateManagerImpl.java [hunk 1]
```diff
@@ -118,7 +125,8 @@ public SwirldStateManagerImpl(
             @NonNull final SwirldStateMetrics swirldStateMetrics,
             @NonNull final TransactionConfig transactionConfig,
             @NonNull final BooleanSupplier inFreeze,
-            @NonNull final State state) {
+            @NonNull final State state,
+            @NonNull final SoftwareVersion softwareVersion) {
```
[the remark this answers is not in the document]
Comment (cody-littley) [reply platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/SwirldStateManagerImpl.java:129]: added

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/SwirldStateManagerUtils.java [hunk 2]
```diff
@@ -38,11 +41,18 @@ private SwirldStateManagerUtils() {}
      * 		object to record stats in
      * @return the newly created state copy
      */
-    public static State fastCopy(final State state, final SwirldStateMetrics stats) {
+    public static State fastCopy(
+            @NonNull final State state,
+            @NonNull final SwirldStateMetrics stats,
+            @NonNull final SoftwareVersion softwareVersion) {
```
[the remark this answers is not in the document]
Comment (cody-littley) [reply platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/SwirldStateManagerUtils.java:47]: added

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/Browser.java [hunk 3]
```diff
@@ -728,6 +733,85 @@ private Collection<SwirldsPlatform> createLocalPlatforms(
...
     }
 
+    /**
+     * Update the address book with the current address book read from config.txt. Eventually we will not do this, and
+     * only transactions will be capable of modifying the address book.
+     *
+     * @param signedState the state that was loaded from disk
+     * @param addressBook the address book specified in config.txt
+     */
+    private static void updateLoadedStateAddressBook(final SignedState signedState, final AddressBook addressBook) {
```
[the remark this answers is not in the document]
Comment (cody-littley) [reply platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/Browser.java:743]: added

**Comment by edward-swirldslabs** [comment]
Running your branch, it looks like the tipset algorithm is turned on and the hashgraph produced is at a throttled rate.   Is this to be expected?

**Review by alittley — APPROVED** [review]

**Pull request merged**